### ISSUE TITLE
NTR: Influenza virus RT-PCR in blood or sputum positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8089]
New term request Influenza virus RT-PCR in blood or sputum positive (nadea3_190227153609): Flu positive; Reverse transcriptase polymerase chain reaction; RNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000737
- label: Positive bloodstream influenza nucleic test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of an influenza virus in the blood circulation by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8089
- synonyms:
